e2e/tests-dfx/ledger.bash

```diff
@@ -14,7 +15,11 @@ setup() {
 
     dfx_start_for_nns_install
 
-    dfx nns install --ledger-accounts 345f723e9e619934daac6ae0f4be13a7b0ba57d6a608e511a00fd0ded5866752 22ca7edac648b814e81d7946e8bacea99280e07c5f51a04ba7a38009d8ad8e89 5a94fe181e9d411c58726cb87cbf2d016241b6c350bc3330e4869ca76e54ecbc
+    dfx extension install nns || true
+    dfx nns install \
```

Comment:
With the recent change to fix this in the extension, can we make a new nns extension release and leave this as it was?

Reply:
Yes that's the plan